integration-cli/docker_cli_plugins_test.go

```diff
@@ -32,7 +32,7 @@ func (s *DockerSuite) TestPluginBasicOps(c *check.C) {
 	id = strings.TrimSpace(id)
 	c.Assert(err, checker.IsNil)
 
-	out, _, err = dockerCmdWithError("plugin", "remove", pNameWithTag)
+	out, _, _ = dockerCmdWithError("plugin", "remove", pNameWithTag)
```

Comment:
@vdemeester I kept `err` and asserted it non-nil. Realized that `dockerCmd` will assume non-nil error...